src/popup/utils/constants.ts

```diff
@@ -1,5 +1,5 @@
 import { SCHEMA } from '@aeternity/aepp-sdk';
```

Comment:
Shouldn't it be removed?

Reply:
Removing `Schema` from codebase is an independent PR